ROADMAP.md

```diff
@@ -1,30 +1,38 @@
-# Roadmap
+# Arena 2019 Roadmap
 
-This document defines a high level roadmap for the arena development.
+This document defines a high level roadmap for Arena development.
 
+### 2019
 
-### 2018
+#### Core CUJs
 
-#### Features
+Objectives: "Make Arena easily to be integrated with External System."
 
-- Enhance Training Job
-  - Move to MPI-operator
-  - Set default CPU/Memory limit according to different types of training: tf-operator, MPI-operator
-  - Support Gang Scheuler
-  - Pytorch-operator
+* Integration
+	* Provide the Java, Python and C++ API for system interaction
+	* Integrate with Pipelines to support Standalone Job, MPI Job, Estimator Job 
 
-- Training History Management
-  - Use CRD to manage the training history
+Objectives: "Simplify the user experience of the data scientists and provide a low barrier to handle different kind of  training jobs and serve different models."
 
-- Integrate with data
-  
-- Muti-tenancy
+* High Level Interfaces:
+	* Submit and manage Flink Jobs
+	* Submit and manage Spark Jobs
```

Comment:
Submit and manage other data processing and machine learning jobs, like Spark, Flink, XDL

Reply:
Updated.